components/dialog/dialog-confirm.js

```diff
@@ -66,12 +73,19 @@ class DialogConfirm extends DialogMixin(LitElement) {
 		`];
 	}
 
-	render() {
-		if (!this._titleId) this._titleId = getUniqueId();
-		if (!this._textId) this._textId = getUniqueId();
```

Comment:
I think we generally put these in the `constructor` so I moved them when adding `offscreenId`, but let me know if there was a reason they were in `render` instead